qualang_tools/addons/variables.py

```diff
@@ -0,0 +1,12 @@
+from qm.qua import *
+
+
+def assign_variables_to_element(element, *variables):
```

Comment:
Also I would add the expected format for "element" and "variables" in the docstring.
Should "variables" be a list (`assign_variables_to_element("qubit_0", [a, b])`) or just add the variables one after the other (`assign_variables_to_element("qubit_0", a, b)`)?

Reply:
Will do.
Currently, it's one after the other `assign_variables_to_element("qubit_0", a, b)`.